modules/osdk-cli-reference-run.adoc

```diff
@@ -7,54 +7,35 @@
 
 The `operator-sdk run` command provides options that can launch the Operator in various environments.
 
-.`run` arguments
+== packagemanifests
+`run packagemanifests` subcommand deploys an Operator's package manifests with OLM. The command argument must be set to a valid package manifest root directory, for example, ``<project-root>/packagemanifests`.
+
+.`packagemanifests` arguments
 [options="header",cols="1,3"]
 |===
 |Arguments |Description
 
+|`--include` (string)
+|The file path to Kubernetes resource manifests, for example, Role, Subscription. These supplement or override defaults generated by run/cleanup.
```

Comment:
nit: Do you mind replacing "for example" with "such as"?  Unfortunately, I'm not an expert in this area, but the initial cap for Role and Subscription makes me think we should either lowercase them as [general references](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-general-references) or use "such as `Role` objects and `Subscription` objects."

Reply:
on second thoughts, I think example might not be needed as `Kubernetes resource manifests` itself is self-explanatory.